multi-node/aws/cmd/kube-aws/command_destroy.go

```diff
@@ -33,6 +36,20 @@ func runCmdDestroy(cmd *cobra.Command, args []string) {
 
 	c := cluster.New(cfg, newAWSConfig(cfg))
 
+	if !force {
+		var confirmation string
+
+		fmt.Print("Are you sure you want to destroy the cluster? This action cannot be undone. Type \"yes\" to proceed: ")
+		if _, err = fmt.Scanln(&confirmation); err != nil {
+			stderr("Failed to read user input: %v", err)
+			os.Exit(1)
+		}
+
+		if confirmation != "yes" {
```

Comment:
My reason for spelling it out completely is to make it slightly safer by making it less likely you'll just quickly type "y<return>" without thinking. While you may create and destroy clusters a lot while using this as a development tool, if kube-aws is intended to be used to manage production clusters, I'd want the program to help protect me from accidentally destroying a cluster.

That said, I'm happy to use y/Y if that's really what you'd prefer.